metricbeat/module/kafka/broker/_meta/docs.asciidoc

```diff
@@ -0,0 +1,20 @@
+This metricset periodically fetches JMX metrics from Kafka Broker JMX.
+
+[float]
+=== Compatibility
+The module has been tested with Kafka 2.1.1. Other versions are expected to work.
```

Comment:
We may add here also a link to the Jolokia module for the compatibility notes there.